### ISSUE TITLE
dead link?

### ISSUE THREAD

**Issue by RayDeanTech** [Azure-Samples/active-directory-javascript-nodejs-webapi-v2#19]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [X ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

The link in the second paragraph of the Overview section appears to be broken:  "JavaScript Single-page application & Node.js web API Tutorial".  Is that supposed to go here:  `ms-identity-javascript-tutorial` ?

Is the intention to use _that_ tutorial alongside _this_ one?
`ms-identity-javascript-tutorial`
`active-directory-javascript-nodejs-webapi-v2`

LONG STORY SHORT:  I've got raw JavaScript on the frontend, and Node on the backend w/MSSQL.... all on premise.

Thanks!
